core/src/test/java/com/linecorp/armeria/client/endpoint/RingHashEndpointSelectionStrategyTest.java

```diff
@@ -0,0 +1,25 @@
+package com.linecorp.armeria.client.endpoint;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import com.linecorp.armeria.client.ClientRequestContext;
+import com.linecorp.armeria.client.Endpoint;
+import com.linecorp.armeria.common.HttpMethod;
+import com.linecorp.armeria.common.HttpRequest;
+
+public class RingHashEndpointSelectionStrategyTest {
+    private static final EndpointGroup emptyGroup = EndpointGroup.of();
+
+    private final ClientRequestContext ctx = ClientRequestContext.of(HttpRequest.of(HttpMethod.GET, "/"));
+
+    @Test
+    void select() {
+        assertThat(EndpointGroup.of(Endpoint.parse("localhost:1234"),
+                                    Endpoint.parse("localhost:2345"))
+                                .selectNow(ctx)).isNotNull();
```

Comment:
Is this selection use `RingHashEndpointSelectionStrategy`? How do users use `RingHashEndpointSelectionStrategy` for their load-balancing strategy? 🤔

Reply:
I think your suggestions would be very convenient for users if implemented.
However, in the existing interface, the size option is not taken into account, which makes the implementation a bit difficult for me.
I would appreciate any ideas on how to incorporate a size option.